2023-07-19 17:05:46,785 INFO    StreamThr :9085 [internal.py:wandb_internal():86] W&B internal server running at pid: 9085, started at: 2023-07-19 17:05:46.784882
2023-07-19 17:05:46,786 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status
2023-07-19 17:05:46,788 INFO    WriterThread:9085 [datastore.py:open_for_write():85] open: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/run-cakuhixp.wandb
2023-07-19 17:05:46,789 DEBUG   SenderThread:9085 [sender.py:send():369] send: header
2023-07-19 17:05:46,790 DEBUG   SenderThread:9085 [sender.py:send():369] send: run
2023-07-19 17:05:47,170 INFO    SenderThread:9085 [dir_watcher.py:__init__():211] watching files in: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files
2023-07-19 17:05:47,171 INFO    SenderThread:9085 [sender.py:_start_run_threads():1100] run started: cakuhixp with start time 1689800746.78329
2023-07-19 17:05:47,171 DEBUG   SenderThread:9085 [sender.py:send_request():396] send_request: summary_record
2023-07-19 17:05:47,171 INFO    SenderThread:9085 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-19 17:05:47,175 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: check_version
2023-07-19 17:05:47,175 DEBUG   SenderThread:9085 [sender.py:send_request():396] send_request: check_version
2023-07-19 17:05:47,229 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: run_start
2023-07-19 17:05:47,235 DEBUG   HandlerThread:9085 [system_info.py:__init__():31] System info init
2023-07-19 17:05:47,235 DEBUG   HandlerThread:9085 [system_info.py:__init__():46] System info init done
2023-07-19 17:05:47,235 INFO    HandlerThread:9085 [system_monitor.py:start():181] Starting system monitor
2023-07-19 17:05:47,235 INFO    SystemMonitor:9085 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-07-19 17:05:47,236 INFO    HandlerThread:9085 [system_monitor.py:probe():201] Collecting system info
2023-07-19 17:05:47,236 INFO    SystemMonitor:9085 [interfaces.py:start():190] Started cpu monitoring
2023-07-19 17:05:47,236 INFO    SystemMonitor:9085 [interfaces.py:start():190] Started disk monitoring
2023-07-19 17:05:47,237 INFO    SystemMonitor:9085 [interfaces.py:start():190] Started gpu monitoring
2023-07-19 17:05:47,238 INFO    SystemMonitor:9085 [interfaces.py:start():190] Started memory monitoring
2023-07-19 17:05:47,238 INFO    SystemMonitor:9085 [interfaces.py:start():190] Started network monitoring
2023-07-19 17:05:47,269 DEBUG   HandlerThread:9085 [system_info.py:probe():195] Probing system
2023-07-19 17:05:47,270 DEBUG   HandlerThread:9085 [gitlib.py:_init_repo():53] git repository is invalid
2023-07-19 17:05:47,270 DEBUG   HandlerThread:9085 [system_info.py:probe():240] Probing system done
2023-07-19 17:05:47,270 DEBUG   HandlerThread:9085 [system_monitor.py:probe():210] {'os': 'Linux-5.15.0-76-generic-x86_64-with-glibc2.31', 'python': '3.10.12', 'heartbeatAt': '2023-07-19T21:05:47.269066', 'startedAt': '2023-07-19T21:05:46.776676', 'docker': None, 'cuda': None, 'args': None, 'state': 'running', 'program': '<python with no main file>', 'host': 'lambda-dual-eureka', 'username': 'robot-lab', 'executable': '/home/robot-lab/anaconda3/envs/BVAE/bin/python', 'cpu_count': 12, 'cpu_count_logical': 24, 'cpu_freq': {'current': 2277.9292499999997, 'min': 2200.0, 'max': 3800.0}, 'cpu_freq_per_core': [{'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 1870.302, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 3800.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2800.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}, {'current': 2200.0, 'min': 2200.0, 'max': 3800.0}], 'disk': {'total': 915.3232879638672, 'used': 865.7034111022949}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 62.715545654296875}}
2023-07-19 17:05:47,270 INFO    HandlerThread:9085 [system_monitor.py:probe():211] Finished collecting system info
2023-07-19 17:05:47,270 INFO    HandlerThread:9085 [system_monitor.py:probe():214] Publishing system info
2023-07-19 17:05:47,270 DEBUG   HandlerThread:9085 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-07-19 17:05:47,271 DEBUG   HandlerThread:9085 [system_info.py:_save_pip():67] Saving pip packages done
2023-07-19 17:05:47,271 DEBUG   HandlerThread:9085 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-07-19 17:05:48,172 INFO    Thread-12 :9085 [dir_watcher.py:_on_file_created():272] file/dir created: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files/requirements.txt
2023-07-19 17:05:48,173 INFO    Thread-12 :9085 [dir_watcher.py:_on_file_created():272] file/dir created: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files/wandb-summary.json
2023-07-19 17:05:48,173 INFO    Thread-12 :9085 [dir_watcher.py:_on_file_created():272] file/dir created: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files/conda-environment.yaml
2023-07-19 17:05:53,173 INFO    Thread-12 :9085 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files/conda-environment.yaml
2023-07-19 17:05:53,304 DEBUG   HandlerThread:9085 [system_info.py:_save_conda():86] Saving conda packages done
2023-07-19 17:05:53,305 INFO    HandlerThread:9085 [system_monitor.py:probe():216] Finished publishing system info
2023-07-19 17:05:53,307 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:05:53,308 DEBUG   SenderThread:9085 [sender.py:send():369] send: files
2023-07-19 17:05:53,308 INFO    SenderThread:9085 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-07-19 17:05:53,654 INFO    wandb-upload_0:9085 [upload_job.py:push():133] Uploaded file /tmp/tmpy327y6w2wandb/rxhhfp59-wandb-metadata.json
2023-07-19 17:05:54,173 INFO    Thread-12 :9085 [dir_watcher.py:_on_file_created():272] file/dir created: /home/robot-lab/Documents/wandb/run-20230719_170546-cakuhixp/files/wandb-metadata.json
2023-07-19 17:05:57,309 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:02,310 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:07,312 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:12,313 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:17,315 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:22,316 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:27,317 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:32,317 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:37,318 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:42,319 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:47,239 DEBUG   SystemMonitor:9085 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-07-19 17:06:47,240 DEBUG   SenderThread:9085 [sender.py:send():369] send: stats
2023-07-19 17:06:48,241 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:53,244 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:06:58,247 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:07:03,250 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:07:08,253 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:07:13,255 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:07:13,503 INFO    StreamThr :9085 [internal.py:wandb_internal():86] W&B internal server running at pid: 9085, started at: 2023-07-19 17:07:13.502866
2023-07-19 17:07:13,504 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status
2023-07-19 17:07:13,506 INFO    WriterThread:9085 [datastore.py:open_for_write():85] open: /home/robot-lab/Documents/wandb/run-20230719_170713-a39livx4/run-a39livx4.wandb
2023-07-19 17:07:13,506 DEBUG   SenderThread:9085 [sender.py:send():369] send: header
2023-07-19 17:07:13,507 DEBUG   SenderThread:9085 [sender.py:send():369] send: run
2023-07-19 17:07:13,633 ERROR   SenderThread:9085 [internal_api.py:execute():324] 404 response executing GraphQL.
2023-07-19 17:07:13,633 ERROR   SenderThread:9085 [internal_api.py:execute():325] {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2023-07-19 17:07:13,633 ERROR   SenderThread:9085 [sender.py:send_run():955] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
Traceback (most recent call last):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 321, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 56, in execute
    request.raise_for_status()
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 953, in send_run
    self._init_run(run, config_value_dict)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 991, in _init_run
    server_run, inserted, server_messages = self._api.upsert_run(
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 1822, in upsert_run
    response = self.gql(
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 293, in gql
    ret = self._retry_gql(
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/util.py", line 946, in check_retry_fn
    return fallback_retry_fn(e)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/util.py", line 888, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-07-19 17:07:17,241 DEBUG   SenderThread:9085 [sender.py:send():369] send: stats
2023-07-19 17:07:18,640 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
2023-07-19 17:07:19,243 DEBUG   HandlerThread:9085 [handler.py:handle_request():144] handle_request: status_report
